src/views/inspector/InspectorViewProvider.ts

```diff
@@ -463,25 +517,21 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
     )) || []
   }
 
-  private async getFeatureEnvironmentStatusesHTML(folder: vscode.WorkspaceFolder) {
+  private getFeatureEnvironmentStatusesHTML() {
     if (this.selectedType !== 'Feature') return ''
 
     function isRecordOfStringEnvironment(obj: any): obj is Record<string, Environment> {
       return typeof obj === 'object' && obj !== null && Object.keys(obj).length > 0
     }
 
-    const featuresCLIController = new FeaturesCLIController(folder)
-    const environmentsCLIController = new EnvironmentsCLIController(folder)
-
-    const featureConfigs = await featuresCLIController.getFeatureConfigurations(this.selectedKey)
-    const environments = await environmentsCLIController.getAllEnvironments()
+    const featureConfigs = this.featureConfigsMap[this.selectedKey]
 
-    if (!isRecordOfStringEnvironment(environments)) {
+    if (!isRecordOfStringEnvironment(this.environments) || !featureConfigs || !featureConfigs.length) {
       return ''
     }
 
-    const html = featureConfigs.map((featureConfig) => {
-      const environment = environments[featureConfig._environment]
+    const html = this.featureConfigsMap[this.selectedKey].map((featureConfig) => {
```

Comment:
seems like this could still be `featureConfigs`, since that was set to `this.featureConfigsMap[this.selectedKey]` above
```suggestion
    const html = featureConfigs.map((featureConfig) => {
```